rviz_rendering/src/rviz_rendering/render_system.cpp

```diff
@@ -205,7 +205,7 @@ RenderSystem::detectGlVersion()
     gl_version_ = force_gl_version_;
   } else {
     Ogre::RenderSystem * renderSys = ogre_root_->getRenderSystem();
-    renderSys->createRenderSystemCapabilities();
+    std::shared_ptr<Ogre::RenderSystemCapabilities>(renderSys->createRenderSystemCapabilities());
```

Comment:
I believe this does not need to be stored. I believe this is a singleton that was returned.

Reply:
I try to fix the issue, but not sure the fixing is correct or not.
Code scan found leaked storage:  renderSys->createRenderSystemCapabilities()

build/rviz_ogre_vendor/ogre-master-ca665a6-prefix/src/ogre-master-ca665a6/OgreMain/include/OgreRenderSystemCapabilities.h
229    RenderSystemCapabilities* GLRenderSystem::createRenderSystemCapabilities() const
230    {
231        RenderSystemCapabilities* rsc = new RenderSystemCapabilities();
444            rsc->addShaderProfile("arbfp1");   

 // Resource rsc is not freed or pointed-to in function addShaderProfile

Ogre::RenderSystemCapabilities::addShaderProfile(Ogre::String const &) does not free or save its parameter this.
599        void addShaderProfile(const String& profile)
600        {
601            mSupportedShaderProfiles.insert(profile);
602
603        }

//addShaderProfile() does not free or save its parameter.